deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml

```diff
@@ -20,14 +20,14 @@ spec:
 
```

Comment:
@ChandanSagar , are there any changes in the values.yaml (additional params) to be added than just the image replacements from 1.0.0

Reply:
For ex, don't see this: https://github.com/helm/charts/blob/master/stable/openebs/values.yaml#L37

The CR template should ideally be the replica of the  values yaml